.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,11 +32,12 @@ $foo->doSomething();
 ```
 
 
-#### BC Breaks/Deprecations
+#### Checklist
```

Comment:
bc breaks and deprecations must go into CHANGELOG.md . other things too, so i changed this section